### PR TITLE
fix(kinput): fix manual change to v-model

### PR DESCRIPTION
**Pull request by kaiarrowood**
# Summary

- Fix issue where manipulating `KInput` `v-model` externally was not reflected
- Wrap `KSelect` in `<div>` for broken markdown issue related to tabbing

## Vue 3

**Did you create a corresponding PR to add this change to the `beta` branch? (required)**

- [x] **Yes**, here is a link to the PR: `[link to beta branch PR]`
- [ ] **No**, I did not create a PR for the `beta` branch and I have added the `port to beta branch` tag to this PR.
- [ ] **No**, because `[type your reasons]`

If you have questions, tag `@adamdehaven` or `@kaiarrowood`.

---

## PR Checklist

- [x] Does not introduce dependencies
- [x] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [x] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [x] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [x] **Framework style:** abides by the essential rules in Vue's style guide
- [x] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [x] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

**Review by HenriPro — COMMENTED** [review]
Fix works! 

TY @kaiarrowood for getting this so fast, that I don't have to commit some messy hack in khcp-ui! 👍

File: docs/components/input.md [hunk 1]
```diff
@@ -215,43 +216,58 @@ You can pass any input attribute and it will get properly bound to the element.
...
 KInput works as regular inputs do using v-model for data binding:
 
-<Komponent :data="{myInput: 'hello'}" v-slot="{ data }">
-  <div>
-    {{ data.myInput }}
-    <KInput
-      v-model="data.myInput"
-      @blur="e => (data.myInput = 'blurred')" />
+<div>
+  {{ myInput }}
```
Comment (HenriPro) [inline docs/components/input.md:220]: Thinking we can take out line 220 and  222 here, as I think they were probably mostly to demo this bug? To me having the clear button is nice however.

**Review by HenriPro — APPROVED** [review]
Fix works! 

TY @kaiarrowood for getting this so fast, that I don't have to commit some messy hack in khcp-ui! 👍

File: docs/components/input.md (hunk 1 above)
Comment (kaiarrowood) [reply docs/components/input.md:220]: Yep, I want to keep this in just to make sure we have the scenario in the docs so we don't break it again!

**Pull request merged**